### ISSUE TITLE
Infinite loop when navigating

### ISSUE THREAD

**Issue by Arneoldenhave** [archanger/LinkRouting#5]
When navigating to "notifications" the router goes into an infinite loop. 

I use swift 6 for the project.

<img width="1022" alt="Screenshot 2024-09-24 at 10 35 41" src="https://github.com/user-attachments/assets/7938ea3b-ad7a-4412-a056-2eeb6061b002">
<img width="420" alt="Screenshot 2024-09-24 at 10 35 59" src="https://github.com/user-attachments/assets/723747f8-2b3c-41bc-8a0f-9dc5c1a59abd">
<img width="594" alt="Screenshot 2024-09-24 at 10 37 06" src="https://github.com/user-attachments/assets/96ee3e20-4ec8-434e-86a5-e1f9c220db22">

**Comment by archanger** [comment]
Yeah, that's the limitation; a `path` passed to `navigator.go(to:)` has to be absolute - `/notifications`

I have in todo to implement relative path navigation, but I cannot guarantee it will be done soon 😞
